### PR TITLE
Update doc add vibration sensor

### PR DESCRIPTION
**Pull request by matrixd2**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/72926
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by matrixd2** [comment]
@frenck it seems that parent PR has been approved, but never get merge, i want to add more devices support. do you have time to check that?

**Comment by frenck** [comment]
Be patient, these things will all be merged before the release. Don't worry

**Comment by matrixd2** [comment]
> Be patient, these things will all be merged before the release. Don't worry

Thanks, can I create new PR for new features before this merged?

File: source/_integrations/yolink.markdown [hunk 1]
```diff
@@ -31,9 +31,10 @@ If you want to use separate credentials, please contact <service@yosmart.com> to
 
 The integration is tested and verified for the following devices from YoLink:
 
-* YS6604
-* YS7103
-* YS7704
-* YS7804/5
-* YS7904
-* YS8003
+* YS6602/4(outlet|plug)
```
Comment (MartinHjelmare) [inline source/_integrations/yolink.markdown]: Please add a space between words. 
```suggestion
* YS6602/4 (outlet | plug)
```

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**